weave-js/src/core/server/remoteHttp.ts

```diff
@@ -252,7 +252,7 @@ export class RemoteHttpServer implements Server {
     const nodes = nodeEntries.map(e => e.node);
     const [payloads, originalIndexes] = this.opts.contiguousBatchesOnly
       ? serializeMulti(nodes)
```

Comment:
I checked and `contiguousBatchesOnly` seems to always be `false`, so I did not convert `serializeMulti`. LMK if I should make a `serialize2Multi`.

Reply:
perhaps add this as a code comment?

Reply:
Added.